pkg/antctl/command_definition.go

```diff
@@ -26,13 +26,14 @@ import (
 
 	"github.com/spf13/cobra"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	apiRuntime "k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/runtime/schema"
 	"k8s.io/klog/v2"
 
 	"antrea.io/antrea/pkg/antctl/output"
 	"antrea.io/antrea/pkg/antctl/runtime"
-	"antrea.io/antrea/pkg/apis/controlplane/v1beta2"
-	"antrea.io/antrea/pkg/controller/networkpolicy"
+	"antrea.io/antrea/pkg/apis/controlplane"
+	endpointServer "antrea.io/antrea/pkg/apiserver/handlers/endpoint"
```

Comment:
ditto, but seems no need to add an alias

Reply:
There is a struct named `Endpoint` in the package, if alias is not created, an error occurs.